pkg/chunk/cached_store.go

```diff
@@ -711,7 +712,11 @@ func NewCachedStore(storage object.ObjectStorage, config Config, reg prometheus.
 			}
 		}()
 	}
+	store.regMetrics(reg)
```

Comment:
Why not put it right after `initMetrics`?

Reply:
it depends on bcache